docs/content/AutoGPT/setup/docker.md

```diff
@@ -71,18 +73,23 @@
           - ./logs:/app/logs
           ## uncomment following lines if you want to make use of these files
           ## you must have them existing in the same folder as this docker-compose.yml
+          ## component configuration file
+          #- ./config.json:/app/config.json
     ```
     </details>
 
 
-4. Download [`.env.template`][.env.template] and save it as `.env` in the AutoGPT folder.
-5. Follow the standard [configuration instructions](../index.md#completing-the-setup),
+1. Download [`.env.template`][.env.template] and save it as `.env` in the AutoGPT folder.
+1. Follow the standard [configuration instructions](../index.md#completing-the-setup),
    from step 3 onwards and excluding `poetry install` steps.
-6. Pull the latest image from [Docker Hub]
+1. Pull the latest image from [Docker Hub]
```

Comment:
I get the idea behind this kind of numbering, but in this case it seems unnecessary since you're only appending a step to the end

Reply:
Reverted